config/config.yml_example_oidc

```diff
@@ -31,6 +31,7 @@ oauth:
   auth_url: https://{yourOktaDomain}/oauth2/default/v1/authorize
   token_url: https://{yourOktaDomain}/oauth2/default/v1/token
   user_info_url: https://{yourOktaDomain}/oauth2/default/v1/userinfo
+  code_challenge_method: S256
```

Comment:
This should be in `config.yml_example` with a bit of documentation and perhaps a link to #303 and maybe oauth.net

I'm curious if you have seen any other usage of `code_challenge_method`?  Is there precedent for `code_challenge_method` and `S256`?